setup/install/providers/aws/aws-ec2.md

```diff
@@ -79,8 +79,24 @@ aws cloudformation deploy --stack-name spinnaker-managed-infrastructure-setup --
 --parameter-overrides AuthArn=FROM_ABOVE ManagingAccountId=FROM_ABOVE --capabilities CAPABILITY_NAMED_IAM --region us-west-2
 ```
 
+## Option-3 : Use AWS Console UI (Manual Steps) 
 
-## Configure Halyard to use AccessKeys (if configured)
+There are 2 options here
+1. Using AWS IAM AccessKey and Secret
+Option number 1 is useful for creation of user with AWS Access Key and secret. This is a common configuration. 
+2. Using AWS IAM Roles
+Option 2 uses the IAM roles *ManagingRole* and *ManagedRoles*. This setting is applied on some environments that have extra security considerations.
+
+## Halyard Configurations
+After the AWS IAM user, roles, policies and trust relationship has been set up, the next step is to add the AWS configurations to spinnaker via Halyard CLI.
+
+The steps are as following:
+* Access the Halyard Pod
```

Comment:
These steps should be an ordered list. Also, can we be explicit about each step, e.g. 

```
Apply the configurations to spinnaker
```

should become

```
4. Apply the configurations to Spinnaker: `hal deploy apply`. 
```

Period at the end of each list item too.